hub-detect/src/main/groovy/com/blackducksoftware/integration/hub/detect/lifecycle/boot/BootManager.java

```diff
@@ -157,13 +144,34 @@ public BootResult boot(DetectRun detectRun, final String[] sourceArgs, Configura
             return BootResult.exit(detectConfiguration);
         }
 
-        Optional<ConnectivityManager> connectivityManagerOptional = determineConnectivity(detectConfiguration, detectOptionManager, detectInfo, gson, objectMapper, eventSystem);
         ConnectivityManager connectivityManager;
```

Comment:
I like that this is all the code for creating the connectivity manager. I think keeping all the flavors of how it gets created together is marvelous.